node/cmd/guardiand/adminclient.go

```diff
@@ -236,13 +236,15 @@ func getPublicRPCServiceClient(ctx context.Context, addr string) (*grpc.ClientCo
 }
 
 func runSignWormchainValidatorAddress(cmd *cobra.Command, args []string) error {
+	ctx := context.Background()
```

Comment:
Here and elsewhere in some other source files (e.g. `message.go`) you create new contexts without managing their lifecycle...what's the reasoning for this? Are we concerned about resource leaks?

Reply:
Thanks for pointing this out. Didn't realise I missed a few. Where possible, I reuse the context from the caller if there is one available.

Reply:
FYI - I fixed this at multiple places, but the specific example you pointed out is actually fine imo. `runSignWormchainValidatorAddress` is a once-off command that is run from the cli.